### ISSUE TITLE
Warnings MultiAnalogSensorsClient

### ISSUE THREAD

**Issue by Nicogene** [robotology/yarp#1666]
Compiling yarp (devel branch) I get warnings like this:
```
/home/ngenesio/yarp/src/devices/multipleanalogsensorsclient/MultipleAnalogSensorsClient.cpp:183: warning: expression result unused [-Wunused-value]
    metadataVector[sens_index].frameName;
    ~~~~~~~~~~~~~~~~~~~~~~~~~~ ^~~~~~~~~
```

It seems a harmless leftover, but I report it anyways, @traversaro feel free to close it if it is not an issue

**Comment by traversaro** [comment]
Thanks @Nicogene , I will be fixing it.

**Comment by Nicogene** [comment]
Closed by #1676
